.travis.yml

```diff
@@ -1,18 +1,23 @@
 language: ruby
 sudo: false
 cache: bundler
-script: 'bundle exec rake test:coverage --trace'
+script: 'bundle exec rubocop && bundle exec rake test:coverage --trace'
```

Comment:
@beauby Not yet. I just activated HoundCI for this project. From the next PR we can revert this diff change.